pep-0675.rst

```diff
@@ -747,13 +737,67 @@ others may consider it in the future, such extensions are out of scope
 for this PEP.
 
 
+Rejected Names
+--------------
+
+We considered a variety of names for the literal string type and
+solicited ideas on `typing-sig
+<https://mail.python.org/archives/list/typing-sig@python.org/thread/VB74EHNM4RODDFM64NEEEBJQVAUAWIAW/>`_. Some
+notable alternatives were:
+
++ ``Literal[str]``: This is a natural extension of the
+  ``Literal["foo"]`` type name but typing-sig `objected
+  <https://mail.python.org/archives/list/typing-sig@python.org/message/2ZQO4NTJEI42KTRJDBL77MNANEXOW7UI/>`_
+  that users could mistake this for the literal type of the ``str``
+  class.
+
++ ``LiteralStr``: This is shorter than ``LiteralString`` but looks
+  weird to the PEP authors.
+
++ ``LiteralDerivedString``: This (along with
+  ``MadeFromLiteralString``) best captures the technical meaning of
+  the type. It represents not just the type of literal expressions
+  such as ``"foo"`` but also that of expressions composed from
+  literals ``"foo" + "bar"``. However, both names seem wordy.
+
++ ``StringLiteral``: Users might confuse this with the existing,
+  unrelated concept of "string literals" in languages like `C++
+  <https://en.cppreference.com/w/cpp/language/string_literal>`_.
+
++ ``SafeString``: While this comes close to our intended meaning, it
+  may mislead users into thinking that the string has been sanitized in
+  some way, perhaps by escaping HTML tags or shell-related special
+  characters.
+
++ ``ConstantStr``: This does not capture the idea of composing literal
+  strings.
+
++ ``StaticStr``: This suggests that the string is statically
+  computable, i.e., without running the program, which is not true. The
+  literal string may vary based on runtime flags, as seen in the
+  `Motivation`_ examples.
+
+Overall, there was no clear winner on typing-sig over a long period,
+so we decided to tip the scales in favor of ``LiteralString``.
+
+
+``LiteralBytes``
+----------------
+
+We could generalize literal byte types such as ``Literal[b"foo"]`` to
+``LiteralBytes``. However, literal byte types are used much less
+frequently than literal string types and we did not find much user
+demand for ``LiteralBytes``, so we decided not to include it in this
+PEP. Others may however consider it in future PEPs.
```

Comment:
```suggestion
demand for ``LiteralBytes``, so we decided not to include it here.
It could, however, be added in a future PEP.
```

Improve phrasing and fix repetition

Reply:
Eh, I prefer the parallelism of the original.

Reply:
This wasn't applied and went stale; I remade it.